Assets/Scripts/Lobby/LobbyManager.cs

```diff
@@ -4,6 +4,19 @@ namespace Lobby
 {
     public class LobbyManager
     {
-        
+        public Unity.Services.Lobbies.Models.Lobby CreateLobby()
+        {
+            var task = Lobbies.Instance.CreateLobbyAsync("MyLobby", 4, new CreateLobbyOptions()
```

Comment:
Would be a bit cleaner to create our lobby options at the beginning of the function rather than doing it here.